src/tests/userRole.test.js

```diff
@@ -168,4 +181,18 @@ describe('/users/role', () => {
         done();
       });
   });
+
+  it('should assign a user a manager', (done) => {
+    chai
+      .request(app)
+      .patch(`/api/v1/user-managements/users/${40}`)
+      .set('token', `Bearer ${token2}`)
+      .send({ managerId: 21 })
+      .end((err, res) => {
+        const { data } = res.body;
+        res.should.have.status(200);
+        expect(data).to.be.an('array');
```

Comment:
@jo-rdan Kindly ascertain values rather than the data type of the returned response.